plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -739,7 +737,7 @@ function OPDSBrowser:browseSearchable(browse_url, username, password)
 end
 
 function OPDSBrowser:onMenuSelect(item)
-    self.catalog_title = item.text or _("OPDS Catalog")
+    self.catalog_title = item.text or _("OPDS catalog")
```

Comment:
I resolved it but I can't seem to push to your repo. Did you disable push access for repo owners when creating the PR?

Anyway, the easiest way to resolve in lieu of command line git is to undo this change:

```suggestion
    self.catalog_title = item.text or _("OPDS Catalog")
```

Then click something like "merge with master" which will probably be an option in the PR once you do that to bring it up to date, and re-apply on the current code.

Reply:
See https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork